simplemonitor/simplemonitor.py

```diff
@@ -263,10 +273,19 @@ def _load_loggers(self, config: EnvironmentAwareConfigParser) -> None:
             new_logger.set_global_info(
                 {"interval": config.getint("monitor", "interval")}
             )
-            module_logger.info(
-                "Adding %s logger %s: %s", logger_type, config_logger, new_logger
-            )
-            self.add_logger(config_logger, new_logger)
+            if new_logger.enabled:
+                module_logger.info(
+                    "Adding %s logger %s: %s",
+                    logger_type,
+                    config_logger,
+                    new_logger.describe(),
+                )
+                self.add_logger(config_logger, new_logger)
+            else:
+                module_logger.info(
+                    "Skipping disabled %s logger: %s", logger_type, config_logger
```

Comment:
I'm happy with not loading disabled loggers/alerters though - I can't immediately think of a reason why that would be an issue elsewhere :)

Reply:
OK, the program does not break with disabled items - see the comments above.